src/popover/src/Popover.js

```diff
@@ -25,6 +25,7 @@ export default class Popover extends Component {
      * When true, the Popover is manually shown.
      */
     isShown: PropTypes.bool,
+    isHover: PropTypes.bool,
```

Comment:
Can you add some documentation to the prop, and maybe we could rename to `triggerOnHover`?